acceptance-tests/src/main/java/me/atam/atam4jsampleapp/ApplicationConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package me.atam.atam4jsampleapp;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.dropwizard.Configuration;
+
+public class ApplicationConfiguration extends Configuration {
+
+    @JsonProperty
+    private int initialDelayInMillis;
```

Comment:
Didn't forget anything but the current situation is a bit confusing.  I will include this in the yaml file as it makes it clearer.  The variable in the yaml file will represent the default value.  This value is actually set by each test using ConfigOverride objects passed to the DropwizardTestSupport constructor.